edx_exams/apps/api/v1/urls.py

```diff
@@ -2,11 +2,20 @@
 
 from django.urls import re_path
 
-from edx_exams.apps.api.v1.views import CourseExamsView
+from edx_exams.apps.api.v1.views import CourseExamsView, ProctoringProvidersView
 from edx_exams.apps.core.constants import COURSE_ID_PATTERN
 
 app_name = 'v1'
 
 urlpatterns = [
-    re_path(fr'exams/course_id/{COURSE_ID_PATTERN}', CourseExamsView.as_view(), name='exams-course_exams'),
+    re_path(
+        fr'exams/course_id/{COURSE_ID_PATTERN}',
+        CourseExamsView.as_view(),
+        name='exams-course_exams'),
+    re_path(
+        r"^providers/available?$",
```

Comment:
I don't think you need `available` in the URL name. I think it's understood that whatever is returned by the API is what's available.

Reply:
ok that makes sense